README.md

```diff
@@ -289,6 +289,7 @@ enforceXPath1 | boolean | Since UiAutomator2 driver version `4.25.0` XPath2 is s
 disableIdLocatorAutocompletion | boolean | According to internal Android standards it is expected that each resource identifier is prefixed with `<packageName>:id/` string. This should guarantee uniqueness of each identifier. Although some application development frameworks ignore this rule and don't add such prefix automatically or, rather, let it up to the developer to decide how to represent their application identifiers. For example, [testTag modifier attribute in the Jetpack Compose](https://developer.android.com/reference/kotlin/androidx/compose/ui/platform/package-summary#(androidx.compose.ui.Modifier).testTag(kotlin.String)) with [testTagsAsResourceId](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary#(androidx.compose.ui.semantics.SemanticsPropertyReceiver).testTagsAsResourceId()) allows developers to set an arbitrary string without the prefix rule. [Interoperability with UiAutomator](https://developer.android.com/jetpack/compose/testing) also explains how to set it. By default UIA2 driver adds the above prefixes automatically to all resource id locators if they are not prefixed, but in case of such "special" apps this feature might be disabled by assigning the setting to `true`.
 includeExtrasInPageSource | boolean | Whether to include `extras` element attribute in the XML page source result. Then, XPath locator can find the element by the extras. Its value consists of combined [getExtras](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#getExtras()) as `keys=value` pair separated by a semicolon (`;`), thus you may need to find the element with partial matching like `contains` e.g. `driver.find_element :xpath, '//*[contains(@extras, "AccessibilityNodeInfo.roleDescription=")]'`. The value could be huge if elements in the XML page source have large `extras`. It could affect the performance of XML page source generation.
+snapshotMaxDepth | int | The number of maximum depth for snapshot. The default value is `70`. This number should be in range [1, 500]. A part of elements source tree might be lost if the value was small. Also, StackOverflow might be caused if the value was large (Issues [1](https://github.com/appium/appium/issues/12545), [2](https://github.com/appium/appium/issues/12892)).
```

Comment:
Could you address the available driver version? The version will be `2.27.0` as `feat`

Reply:
Thank you for your review!
I have a question. I added a setting API `snapshotMaxDepth` in appium-uiautomator2-server. I think a setting API does not depend on the driver version, I'm sorry if I misunderstand.

Reply:
Correct for the UIA2 server, but users use UIA2 driver to manage UIA2 server as well. It means users who install a new pushed driver version with `5.12.0` UIA2 server version can use this `snapshotMaxDepth`. So to expose this `snapshotMaxDepth` for users via UIA2 server, they need a new UIA2 driver pushed by this PR merge

Reply:
...part of the elements source tree...

...if the value is too low...

...if the value is too high (issues [12545](https://github.com/appium/appium/issues/12545), [12892](https://github.com/appium/appium/issues/12892))...

Reply:
Thanks! I added the available version and updated some commented points. https://github.com/appium/appium-uiautomator2-driver/pull/629/commits/c5f5525c77a318a1ae529ac85a54ffd97e75a5c2